### DISCUSSION TITLE
ProductSearchModel | SFRA | fetch only recently modified products

### DISCUSSION THREAD

**Discussion by asagar3** [SalesforceCommerceCloud/sfcc-ci#419] [Team Posts]
I am trying to customized SFMC OOB catalog export. Is there any way in SFRA to get only last modified products in search result using ProductSearchModel ? 

var psm = new ProductSearchModel();
psm.getOrderableProductsOnly();
psm.search();

Is there any API ? so that search() returns only last modified products after last exported products in catalog export job.

**Comment by pravngaur** [comment]
Hello Anju,

There is nothing such ootb.

What you can do is create a custom attribute at product level say
isExported.

While exporting, your job will set this flag to true and during import all
the product updates will set this flag to true.

Now, its straight forward to use this additional attribute as search
refinement while using the search api.

Pay caution as this could have side effects on job run time, depending upon
your catalog and feed size, for a moderate sized catalog it should be good.

Hope this helps.
